test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/ActivityExtensionsTest.cs

```diff
@@ -1,303 +1,307 @@
-            bool endCalled = false;
-
-            testActivityProcessor.StartAction =
-                (a) =>
-                {
-                    startCalled = true;
-                };
-
-            testActivityProcessor.EndAction =
-                (a) =>
-                {
-                    endCalled = true;
-                };
-
-            var openTelemetrySdk = OpenTelemetrySdk.EnableOpenTelemetry(b => b
-                            .AddActivitySource(ActivitySourceName)
-                            .UseOtlpExporter(
-                                null, p => p.AddProcessor((next) => testActivityProcessor)));
-
-            var source = new ActivitySource(ActivitySourceName);
-            var activity = source.StartActivity("Test Otlp Activity");
-            activity?.Stop();
-
-            Assert.True(startCalled);
-            Assert.True(endCalled);
-        }
-
-        private static void AssertActivityTagsIntoOtlpAttributes(
-            List<KeyValuePair<string, object>> expectedTags,
-            RepeatedField<OtlpCommon.KeyValue> otlpAttributes)
-        {
-            Assert.Equal(expectedTags.Count, otlpAttributes.Count);
-            for (var i = 0; i < expectedTags.Count; i++)
-            {
-                Assert.Equal(expectedTags[i].Key, otlpAttributes[i].Key);
-                AssertOtlpAttributeValue(expectedTags[i].Value, otlpAttributes[i]);
-            }
-        }
-
-        private static void AssertOtlpAttributes(
-            List<KeyValuePair<string, object>> expectedAttributes,
-            RepeatedField<OtlpCommon.KeyValue> otlpAttributes)
-        {
-            Assert.Equal(expectedAttributes.Count(), otlpAttributes.Count);
-            for (int i = 0; i < otlpAttributes.Count; i++)
-            {
-                Assert.Equal(expectedAttributes[i].Key, otlpAttributes[i].Key);
-                AssertOtlpAttributeValue(expectedAttributes[i].Value, otlpAttributes[i]);
-            }
-        }
-
-        private static void AssertOtlpAttributeValue(object originalValue, OtlpCommon.KeyValue akv)
-        {
-            switch (originalValue)
-            {
-                case string s:
-                    Assert.Equal(s, akv.Value.StringValue);
-                    break;
-                case bool b:
-                    Assert.Equal(b, akv.Value.BoolValue);
-                    break;
-                case long l:
-                    Assert.Equal(l, akv.Value.IntValue);
-                    break;
-                case double d:
-                    Assert.Equal(d, akv.Value.DoubleValue);
-                    break;
-                default:
-                    Assert.Equal(originalValue.ToString(), akv.Value.StringValue);
-                    break;
-            }
-        }
-    }
-}
+                });
+
+            // This following is done just to set Resource to Activity.
+            using var openTelemetrySdk = OpenTelemetrySdk.EnableOpenTelemetry(b => b

```

Comment:
only change in this file is this line. 
Visual Studio fixed file endings so entire file appears to be changed!